Formula/libvdpau.rb

```diff
@@ -17,7 +17,7 @@ class Libvdpau < Formula
   # Required
   depends_on "linuxbrew/xorg/dri2proto" => :build
   depends_on "linuxbrew/xorg/libxext"
-  depends_on "linuxbrew/xorg/libx11"
+  depends_on "linuxbrew/xorg/libx11" => :build
```

Comment:
Is it needed at all? Is it used only for its headers, or is it statically linked?

Reply:
The same question goes for each of these.

Reply:
built libvdpau from source: https://gist.github.com/86c47f853abf03cc35d78653ead6ec97

Reply:
So, `libvdpau` does not depend on `libx11` then?

Reply:
1. In `/home/linuxbrew/.linuxbrew/Cellar/libvdpau/1.1.1/include/vdpau/vdpau_x11.h` there is  `#include <X11/Xlib.h>`
2. In `/home/linuxbrew/.linuxbrew/Cellar/libvdpau/1.1.1/lib/pkgconfig/vdpau.pc` there is `Requires.private: x11`

So, it needs libx11, it's just not linked to it... Does this secure a standard dependency type?

Reply:
That's pretty common for some libraries, like `boost`. It's standard `:build` dependency.